src/adapter/src/catalog.rs

```diff
@@ -3274,7 +3275,8 @@ impl Catalog {
         c: &Catalog,
     ) -> Result<Catalog, Error> {
         let mut c = c.clone();
-        let mut awaiting_dependencies: BTreeMap<GlobalId, Vec<_>> = BTreeMap::new();
+        let mut awaiting_id_dependencies: BTreeMap<GlobalId, Vec<_>> = BTreeMap::new();
```

Comment:
yeah, someone from @MaterializeInc/surfaces needs to look at this commit :sweat_smile: